src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SimplifyBooleanReturnCheck.java

```diff
@@ -51,6 +51,41 @@
  * <pre>
  * &lt;module name=&quot;SimplifyBooleanReturn&quot;/&gt;
  * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * public class Test {
+ *
+ *  private boolean cond;
+ *  private Foo a, b;
+ *
+ *  public boolean check1() {
+ *  if (cond)
+ *    return true;
+ *  else
+ *    return false;
+ *  }
+ *  // violation, could be simplified
```

Comment:
Please put violation comment on line where violation is reported

Reply:
Please do not resolve comments, reviewers should do it.

In this case comment is not on the line where violation occurs, but before method